stdlib/public/core/Equatable.swift

```diff
@@ -247,10 +247,10 @@ extension Equatable {
 ///   - lhs: A reference to compare.
 ///   - rhs: Another reference to compare.
 @inlinable // trivial-implementation
-public func === (lhs: AnyObject?, rhs: AnyObject?) -> Bool {
+public func ===<T: AnyObject, U: AnyObject>(lhs: T?, rhs: U?) -> Bool {
```

Comment:
This isn't a full isomorphism right? because you could have two things stated as `let t: AnyObject = ...` and pass that as `===`.

Reply:
Ah, yes, you're right. Actually some lit tests show this problem, too (on regular Swift), so this is a breaking change. I changed it to #if based alternative implementation only for embedded Swift.